pages/login.vue

```diff
@@ -0,0 +1,7 @@
+<template>
+    <div class="login">
+        <TextIamgeTransparent image="/images/gamers.jpg"></TextIamgeTransparent>
```

Comment:
I suggest renaming this component to `TextImageTransparent`, so without the typo